testground/README.md

```diff
@@ -0,0 +1,7 @@
+# Testground
+
+This docker image is used for testing of OpenDAX-v4
```

Comment:
I'd rather write that this image is used as the runtime for running Testground tests in general